docs/geedocs/index.html

```diff
@@ -13,21 +13,8 @@
   <div class="content"> <a name="top_of_file"></a>
     <p><img src="art/common/googlelogo_color_260x88dp.png" width="130" height="44" alt="Google logo" /></p>
     <h2>Google Earth Enterprise  Documentation</h2>
-    <p>This documentation contains legacy information that may not apply to the
-       Google Earth Enterprise Open Source version. This bundle is also 
-       available in the GEE Server 5.2.0 installation package.</p>
-    <h4><a href="answer/2995465.html">Get started</a></h4>
-    <h4><a href="answer/4412440.html">Fusion tutorial</a></h4>
-    <h4><a href="answer/3468016.html">Install Google Earth Enterprise</a></h4>
-    <h4><a href="answer/3481464.html">Fusion administration</a></h4>
-    <h4><a href="answer/6153619.html">Fusion resources and projects</a></h4>
-    <h4><a href="answer/3468004.html">GEE Server administration</a></h4>
-    <h4><a href="answer/173056.html">GEE Server configuration and security</a></h4>
-    <h4><a href="answer/6006981.html">Google Earth Enterprise client</a></h4>
-    <h4><a href="answer/4580164.html">Portable</a></h4>
-    <h4><a href="answer/6014171.html">Developers</a></h4>
-    <h4><a href="answer/6121524.html">Troubleshooting and error messages</a></h4>
-    <h4><a href="answer/7159245.html">Release notes</a></h4>
+    <h3><a href="./5.2.0/index.html">5.2.0</a></h3>
+    <h3><a href="./5.1.3/index.html">5.1.3</a></h3>
```

Comment:
Some thoughts about this:

- I don't think we need 5.1.3 documents.  Starting with 5.2.0 should be enough.
- We also need a folder for the next release (the one under development).  I suppose we could make that a part of the release process - copy the existing release docs into a folder for the new release.  Maybe that's the intention.  When we do that, we should modify the link for the release under development to indicate that it's under development.
- The same docs are installed when you install fusion and server.  We need to test that it still works in that case.  It would be great if it only installs the latest version of the docs.  Perhaps updating the link at earthenterprise/earth_enterprise/landing_page/manual to point to the docs for the latest release would do that.  We'd just need to update the release instructions.
- As Asmita noted, we need to get the release instructions into the wiki at some point.
- Asmita also pointed out that it would be great to have a way to avoid duplication and only store the differences between releases.  That doesn't have to be done right now.